server.go

```diff
@@ -2821,7 +2822,8 @@ func (s *server) Peers() []*peer {
 // form "#RRGGBB", parses the hex color values, and returns a color.RGBA
 // struct of the same color.
 func parseHexColor(colorStr string) (color.RGBA, error) {
-	if len(colorStr) != 7 || colorStr[0] != '#' {
+	// Check if the hex color string is a valid color representation
+	if res, _ := regexp.MatchString(`^#[A-Fa-f0-9]{6}$`, colorStr); res == false {
```

Comment:
what's the rationale for not handling the error value?

Reply:
I see, error value is returned if the regex doesn't compile. to get around this, we can consider using `MustCompile`https://golang.org/pkg/regexp/#MustCompile to safely initialize the pattern